gradle.properties

```diff
@@ -1,4 +1,4 @@
-projectVersion=5.5.2-SNAPSHOT
+projectVersion=6.0.0-SNAPSHOT
```

Comment:
Did you change it to the next major because we no longer exposed these managed versions?

Reply:
> Did you change it to the next major because we no longer exposed these managed versions?

Yes, see my note in the description. I am erring on the side of caution here.